web/styles/rendered_markdown.css

```diff
@@ -23,15 +23,31 @@
         margin-left: 0;
     }
 
-    /* Ensure ordered lists are nicely centered in 1-line messages */
+    /* Ensure ordered lists are nicely centered in 1-line messages for upto 2
```

Comment:
"up to" are separate words :)

Reply:
Also I think this comment might be better moved to just before the `::before` block that implements this part of the system? The first couple bits of styling for `ol` elements feel unrelated.

Reply:
Done, this is ready for another look @timabbott :)